packages/forma-36-react-components/src/components/Button/Button.tsx

```diff
@@ -16,12 +16,12 @@ const styles = require('./Button.css');
 
 export type ButtonProps = {
   icon?: IconType;
-  indicateDropdown?: boolean;
+  hasDropdown?: boolean;
```

Comment:
I'd call it `withDropdownCaret` or something like this

Reply:
Nice, that's a lot clearer - `hasDropdown` is pretty misleading for this one isn't it. I'll update.

Reply:
Do you want to add "with" as an allowed prefix for booleans in the component principles or shouldn't it rather be `hasDropdownCaret`?

Reply:
Yup! That's exactly what we went with - the prop is now `hasDropdownCaret` (your comment is on an outdated diff). I'll mark the conversation as resolved